src/Components/Home.jsx

```diff
@@ -21,28 +22,24 @@ const Home = () => {
           count += 1;
           return (
             <li id={data.volumn} key={count}>
+              <NavLink to={`/details/:${data.symbol}/:${count}`}>
+                <button type="button" className="Button input-submit" onClick={() => dispatch(IC(data.symbol))}><FaAngleRight /></button>
+              </NavLink>
               <h2>{data.symbol}</h2>
               <h3>{data.companyName}</h3>
               <div className="Fetch-footer">
-                <div>
-                  <p>
-                    <span className="bold">Price:</span>
-                    {' '}
-                    {data.price}
-                  </p>
-                </div>
-                <div className="align">
-                  <p>
-                    {' '}
-                    <span className="bold">Industry:</span>
-                    {' '}
-                    {data.industry}
-                  </p>
-                </div>
+                <p>
+                  <span className="bold">
+                    Price:
+                  </span>
+                  $
+                  {data.price}
+                  <span className="bold">
+                    Industry:
+                  </span>
+                  {data.industry}
+                </p>
```

Comment:
- [ ] Kindly work and improve the look of the app on the home page to match the design given.

![original](https://user-images.githubusercontent.com/53124548/217783603-46da05e9-5a56-41d5-8759-548d3f383d0c.png)

Reply:
Well resolved ✔️